src/ui/app/App.js

```diff
@@ -175,12 +175,13 @@ import PageNotFoundContainer from 'ui/app/PageNotFoundContainer';
 import CloneWidgetPage from 'ui/widgets/clone/CloneWidgetPage';
 import EmailConfigPage from 'ui/email-config/EmailConfigPage';
 
-import InternalPage from 'ui/internal-page/InternalPage';
+import PageLayout from 'ui/app/PageLayout';
 import RowSpinner from 'ui/pages/common/RowSpinner';
 import entandoApps from 'entando-apps';
 import AboutPage from 'ui/about/AboutPage';
 import LicensePage from 'ui/license/LicensePage';
 import getRuntimeEnv from 'helpers/getRuntimeEnv';
+import InternalPage from 'ui/internal-page/InternalPage';
 
 const appsRoutes = entandoApps.reduce((routes, app) => (
   [
```

Comment:
on line 194, we are still using InternalPage as a wrapper on a render method, shouldn't it be replaced by the PageLayout as well?